lib/rubocop/cop/style/redundant_conditional.rb

```diff
@@ -25,26 +25,33 @@ module Style
       #   # good
       #   x != y
       class RedundantConditional < Cop
+        include AutocorrectAlignment
+
         COMPARISON_OPERATORS = RuboCop::AST::Node::COMPARISON_OPERATORS
 
         MSG = 'This conditional expression can just be replaced by %s'.freeze
 
         def on_if(node)
           return unless offense?(node)
 
-          add_offense(node,
-                      :expression,
-                      format(MSG, replacement_condition(node)))
+          add_offense(node, :expression)
         end
 
         private
 
+        def message(node)
+          replacement = replacement_condition(node)
+          msg = node.elsif? ? "\n#{replacement}" : replacement
+
+          format(MSG, msg)
+        end
```

Comment:
I think this message is a bit difficult to understand.


![170831182915](https://user-images.githubusercontent.com/4361134/29916599-2f3499c0-8e7a-11e7-83ba-b72944432faa.png)

the `else\n x ==y` is confusing. It also looks like code.
I guess it is a little easy to understand by adding backtick around `%s`( https://github.com/bbatsov/rubocop/pull/4681/files#diff-c73e58428562c0f62526309365c75b87R32 ).

Reply:
Thanks. Actually, I tried to use backticks, but found out that it does not work with multiline strings, since [RuboCop::Formatter::SimpleTextFormatter#annotate_message](annotate_message) uses single-line regex for replacement purposes:

```ruby
def annotate_message(msg)
  msg.gsub(/`(.*?)`/, yellow('\1'))
end
```

Even though the fix is trivial (we just need to add the `m` modifier to the regex), should we do it in the scope of this pull request?

Reply:
> Even though the fix is trivial (we just need to add the m modifier to the regex), should we do it in the scope of this pull request?

Yes, I think it should be fixed in this pull-request. Because the change is very small, and the change is necessary to display the message correctly.

Reply:
Makes sense, done.